### DISCUSSION TITLE
matchmaking

### DISCUSSION THREAD

**Discussion by kunseru** [LuisMayo/vhs-alternative-server#56] [Q&A] [answered]
will matchmaking ever be a thing like a queue system that creates a private lobby and people that press on queue normally join that lobby or no?

**Comment by SkelXton** [comment, marked as answer]
So, when the studio shut down the dedicated servers, the MMS was moved to connect players through a p2p lobby. It even sends a lobby code when people join 😆 

I believe Luigi does have the things necessary to complete a backend for matchmaking, however with how few players we see playing at a given time in addition to the extra overhead that supporting progression data could cause, we deemed it extra work for not much more benefit. The games would function similarly to customs but have fewer options (and no bots to backfill lobbies). 

Could it happen? Yeah, Luigi might do it, but as far as I know, there aren't any plans for it at the moment.

**Comment by LuisMayo** [comment]
Hi!

@SkelXton is right. Matchmaking is possible as in we have the required network capture data to remake it. However given the low amount of users playing this game and the fact that matchmaking is a bit harder to code, I have decided not to tackle on this issue right now. Specially because even if we get matchmaking working form a technical standpoint, the low playerbase makes it so matches probably won't be happening anyway.

That being said the project is open source, anyone with the required technical expertise is welcomed to try and I'll try to help them
